openspoor/visualisations/spoorkaart.py

```diff
@@ -0,0 +1,220 @@
+import pandas as pd
+import folium
+from shapely.geometry import point
+from typing import List, Optional, Dict, Tuple
+from pathlib import Path
+from ..utils.common import read_config
```

Comment:
relatieve imports gaan snel fout. "from openspoor.utils.common import read_config" is netter. Zie wel dat het in alle files nu zo staat..